platform/osgi/platform-osgi-configadmin/src/test/java/org/codice/felix/cm/file/CopiedEncryptionServiceTest.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.felix.cm.file;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.keyczar.Crypter;
+import org.keyczar.exceptions.KeyczarException;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CopiedEncryptionServiceTest {
```

Comment:
❗️ If we're copying the source of the impl, I think we should be doing 1 of 3 things:
 1. Not unit testing it at all
 2. Copying over the original unit tests
 3. Cleaning up the original unit tests THEN copying those over

My personal preference would be for option 2.

Reply:
@lessarderic, what do you think about this?

Reply:
Instead just embedded the encryption service impl jar to cut down on the code